docs/en/sql-reference/functions/other-functions.md

```diff
@@ -3688,3 +3720,118 @@ Result:
 ```response
 {'version':'1','serial_number':'2D9071D64530052D48308473922C7ADAFA85D6C5','signature_algo':'sha256WithRSAEncryption','issuer':'/CN=marsnet.local CA','not_before':'May  7 17:01:21 2024 GMT','not_after':'May  7 17:01:21 2025 GMT','subject':'/CN=chnode1','pkey_algo':'rsaEncryption'}
 ```
+
+## lowCardinalityIndices
+
+For each row in the current [block](../../development/architecture.md/#block-block), returns the index of the value in the dictionary of unique values for columns of [LowCardinality](../data-types/lowcardinality.md) type. 
```

Comment:
Let's replace l. 3726-3730 by:

```
Returns the position of a value in the dictionary of a [LowCardinality](../data-types/lowcardinality.md) column.
Positions start at 1.
Since LowCardinality have per-part dictionaries, this function may return different positions for the same value in different parts.
```